tests/framework/extensions/namespaces/deletenamespaces.go

```diff
@@ -0,0 +1,8 @@
+package namespaces
+
+import v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+
+// DeleteNamespace is a helper function which deletes a namespace by the ID and returns the response obtained.
+func DeleteNamespace(namespaceID *v1.SteveAPIObject, steveclient *v1.Client) error {
+	return steveclient.SteveType(NamespaceSteveType).Delete(namespaceID)
```

Comment:
Anu is this even necessary? Why not just make the Steve client call?

Reply:
Sure, I think previously when this was introduced in the rbac_test.go, it was suggested we move this to rbac.go helper file. and made this an extension but agree that it makes sense not to have an extension for just a single line.